released/packages/coq-infotheo/coq-infotheo.0.2.1/opam

```diff
@@ -0,0 +1,52 @@
+# This file was generated from `meta.yml`, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+version: "dev"
```

Comment:
```suggestion
opam-version: "2.0"
maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
```

Reply:
I see. Thank you. This is all the more appropriate that I actually edited the file manually because I could not figure out how to change the `build` field.